.env.example

```diff
@@ -70,9 +70,8 @@ export WEB_CONCURRENCY=1
 #export WEB_RELOAD=false
 export WEB_RELOAD=true
 
-# GUNICORN_TIMEOUT sets the timeout for gunicorn workers, defaults to 120 
-# seconds.
-#export GUNICORN_TIMEOUT=
+# Configure the timeout value in seconds for gunicorn.
+#export WEB_TIMEOUT=
```

Comment:
Given it defaults to 120, can you also set `export WEB_TIMEOUT=120` here so folks know what the default value is?

Reply:
Sure thing.